src/main/java/com/iexec/common/chain/eip712/EIP712Entity.java

```diff
@@ -43,7 +45,7 @@ protected EIP712Entity(EIP712Domain domain, M message) {
 
     @Override
     public HashMap<String, List<TypeParam>> getTypes() {
```

Comment:
We can use map everywhere
(you can add a todo if you prefer)

Reply:
Thanks, I wondered whether I could use a `Map` there. That'll make things cleaner.